development/extensions/skeleton_extension.rst

```diff
@@ -30,17 +30,17 @@ installed into a phpBB board just the same as any other.
 
 .. note::
 
-    The Skeleton Extension is a tool for extension developers. As such,
-    it is intended to be installed in a phpBB board used for
-    offline/local development of extensions. It is not necessarily
-    intended to be installed on live/production web sites.
+    The Skeleton Extension is intended to be installed in a phpBB
+    board used for offline/local development of extensions. It is
+    not necessarily intended to be installed on live/production web
```

Comment:
Just because it's not intended to be used on production sites doesn't mean it can't. One of our own Jr Team members uses it on a live site which is devoted to phpBB developers and extension developers. In that scenario, it is right at home.

The point is this is for developers. This is not for "regular" phpBB boards. It's not really meant to be pretty, and its why it also has a CLI componenet. It's a tool, not a feature. The text above is meant to drive this point home with a lot less words (hehe) that this is for those who know what it is and have a use for it.